### PR TITLE
Address documentation misalignment 

### PR DESCRIPTION
**Pull request by galanCA**
## Description

Address missing CSE fields in documentation or vise versa.

## Author Progress Checklist:

- [x] Open draft pull request
    - [x] Make title clearly understandable in a standalone change log context
    - [x] Assign yourself the issue
    - [x] Add at least one label
- [ ] Make code changes (if you haven't already)
- [x] Self-review of code
    - [ ] My code follows the style guidelines of this project
    - [ ] I have performed a self-review of my code
    - [ ] I have added comments to my code, particularly in hard-to-understand areas
    - [ ] I have only committed the necessary changes for this fix or feature
    - [x] I have made corresponding changes to the documentation
    - [x] My changes generate no new warnings
    - [ ] I have ensured that my fix is effective or that my feature works as intended by:
        - [ ] exercising the code changes in the test framework, and/or
        - [ ] manually verifying the changes (as explained in the pull request description above)
    - [ ] My changes pass all local tests
    - [ ] My changes successfully pass CI checks
- [ ] Move pull request out of draft mode and assign reviewers
- [ ] Iterate with reviewers until all changes are approved

## Reviewer Checklist:

 - [x] Read the pull request description
 - [x] Perform a code review on GitHub
 - [x] Confirm all CI checks pass and there are no build warnings
 - [x] Pull, build, and run automated tests locally
 - [x] Perform manual tests of the fix or feature locally
 - [x] Add any review comments, if applicable
 - [x] Iterate with author until all changes are approved
 - [x] If you are the last reviewer to approve, merge the pull request and delete the branch

### CODE REVIEW

File: doc/src/records/dhwheater.md [hunk 1]
```diff
@@ -233,43 +267,6 @@ Water heater surround temperature, used only in detailed HPWH models (whHeatSrc=
 
 Air source heat pump type, valid only if whHeatSrc=ASHPX. These choices are supported by the detailed HPWH model.  Except for Generic, all heater characteristics are set by HPWH based on whASHPType.
 
-<%= csv_table(<<END, :row_header=> true)
```
Comment (nealkruis) [inline doc/src/records/dhwheater.md:236]: Why was this removed?

File: doc/src/records/dhwheater.md [hunk 2]
```diff
@@ -85,6 +85,28 @@ available for electric (air source heat pump and resistance) SMALLSTORAGE water
   ----------- ---------------------- ------------- -------------- -----------------
                *Codes listed above*   FUEL          No             constant
 
+**whResType=*choice***
+
+Resistance heater type, valid only if wh_heatSrc=_ELRESX, else ignored. These choices are supported by the detailed HPWH model.  Except for Generic, all heater characteristics are set by HPWH based on whResType.
```
Comment (nealkruis) [inline doc/src/records/dhwheater.md:90]: Some of the comments in CNRECS refer to other CNRECS field names, but in the user documentation they should refer to other CUL names.

File: doc/src/records/dhwheater.md [hunk 3]
```diff
@@ -98,6 +120,18 @@ Storage tank volume. Must be omitted or 0 for instantaneous whTypes.  Used by HP
               runtime errors) 
   -----------------------------------------------------------------------
 
+**whVolRunning=*float***
+
+Running storage volume = vol above aquastat. Determines require total volume based on aquastat position. Ecotope's HPWH tank and heater.
```
Comment (nealkruis) [inline doc/src/records/dhwheater.md:125]: Because this is user facing documentation, we should avoid unnecessary abbreviations like "vol". There are a lot of these in the CNRECS comments, but they should be spelled out here (with exceptions for well known or previously defined acronyms in the same file).

File: doc/src/records/dhwsolarcollector.md [hunk 4]
```diff
@@ -70,18 +70,117 @@ Fit y-intercept
...
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "0.72",
+  required: "No",
+  variability: "constant") %>
+
+**scOprMassFlow=*float***
+
+Collector loop operating mass flow rate
```
Comment (nealkruis) [inline doc/src/records/dhwsolarcollector.md:97]: Let's end each description with a period.

File: doc/src/records/rsys.md [hunk 5]
```diff
@@ -257,6 +327,237 @@ For rsType=ASHP, rated heating coefficient of performance at outdoor dry-bulb te
...
+<%= member_table(
+  units: "",
+  legal_range: "x $>$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "Before set up or at the end of interval") %>
+
+**rsTypeAuxH=*choice***
+
+Type of auxiliary heat. The functions have C_AUXHEATTY as a prefix.
```
Comment (nealkruis) [inline doc/src/records/rsys.md:541]: Translate choice type enumerations to the corresponding user input choice.

File: doc/src/records/dhwsys.md [hunk 6]
```diff
@@ -144,11 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
+**wsDWashrDrawWaste=*float***
+
+Dish washer water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:288]: The content below was general documentation for a set of inputs. It should be preserved.

File: doc/src/records/dhwsys.md [hunk 7]
```diff
@@ -186,15 +323,71 @@ Day waste scaling factor.
...
+**wsDWashrDayWasteF=*float***
+
+Relative dish washer water draw for day of waste scheme.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "constant") %>
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:390]: Again, there is left-over documentation from the set of inputs below.

File: doc/src/records/dhwsys.md [hunk 8]
```diff
@@ -227,6 +420,94 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
...
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "wsTUse",
+  required: "No",
+  variability: "constant") %>
+
+**wsVolRunningDes=*float***
+
+Running volume for design.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:436]: You are missing a lot of the content from CNRECS.

File: doc/src/records/dhwsys.md [hunk 9]
```diff
@@ -227,6 +420,94 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
...
+  required: "No",
+  variability: "constant") %>
+
+**wsDRMethod=*choice***
+
+Response control method for domestic hot water demand.
+
+<%= csv_table(<<END, :row_header => true)
+Choice, Description
+C_DHWDRMETH_NONE, None
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:473]: Valid user input for choices is found in CNDTYPES.DEF. Look for the choice name between the underbars (e.g., DHWDRMETH).

File: doc/src/records/dhwsys.md [hunk 10]
```diff
@@ -227,6 +420,94 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
...
+  required: "No",
+  variability: "At the start of a run") %>
+
+**wsDRSignal=*choice***
+
+Response control signal for domestic hot water demand.
+
+<%= csv_table(<<END, :row_header => true)
+Choice
+C_DHWDRSIG_ON
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:491]: Fix user choice options.

File: doc/src/records/gain.md [hunk 11]
```diff
@@ -4,6 +4,20 @@ A GAIN object adds sensible and/or latent heat to the ZONE, and/or adds arbitrar
...
+
+# Gain Modeling in <% if inactive_CNE_records %>CNE <% end %>zones
+
+<% if inactive_CNE_records %>In the CNE zone mode, t<% else %>T<% end %>he radiant internal gain is distributed to the surfaces in the zone, rather than going directly to the zone "air" heat capacity (znCAir). A simple model is used -- all surfaces are assumed to be opaque and to have the same (infrared) absorptivity -- even windows. Along with the assumption that the zone is spherical (implicit in the current treatment of solar gains), this allows distribution of gains to surfaces in proportion to their area, without any absorptivity or transmissivity calculations. The gain for windows and quick-model surfaces is assigned to the znCAir, except for the portion which conducts through the surface to the other side rather than through the surface film to the adjacent zone air; the gain to massive (delayed-model) surfaces is assigned to the side of surface in the zone with the gain.
```
Comment (nealkruis) [inline doc/src/records/gain.md:16]: Is this the same content as the paragraph above? Let's talk over the best way to handle this section.

File: doc/src/records/report.md [hunk 12]
```diff
@@ -196,6 +196,10 @@ Specifies meter(s) to be reported, for *rpType*=MTR.
   ----------- ----------------------------- ------------- --------------------------- -----------------
               name of a *METER*, ALL, SUM                 Required for *rpType*=MTR   constant
 
+**rpAH**
```
Comment (nealkruis) [inline doc/src/records/report.md:199]: Did you see this included already in the conditional statement below?

File: doc/src/records/rsys.md [hunk 13]
```diff
@@ -274,6 +591,17 @@ For rsType=ASHP, auxiliary electric (“strip”) heating capacity. If AUTOSIZEd
   ----------- --------------------------- ------------- -------------- -----------------
   Btu/hr      *AUTOSIZE* or *x* $\ge$ 0   0             no             constant
 
+**rsAFUEAuxH=*float***
+
+Auxiliary furnace heating for annualized fuel utilization efficiency.
```
Comment (nealkruis) [inline doc/src/records/rsys.md:596]: See the previous documentation that you deleted before (from `rsCOPAuxH`). I think that was the same input, but with the wrong name.

File: doc/src/records/top-members.md [hunk 14]
```diff
@@ -287,6 +273,24 @@ NONE            Disable advanced shading calculations
   ----------- ----------------------- -------------- -------------- -----------------
               *Choices above*          PENUMBRA           No            constant
 
+**slrInterpMeth=*choice***
+
+Solar interpolation method.
+
+<%= csv_table(<<END, :row_header => true)
+Choice
+C_SLRINTERPMETH_CSE
```
Comment (nealkruis) [inline doc/src/records/top-members.md:282]: Convert to user input choices.

File: doc/src/records/rsys.md [hunk 15]
```diff
@@ -266,6 +336,253 @@ For rsType=ASHP, rated heating coefficient of performance at outdoor dry-bulb te
...
+  required: "No",
+  variability: "constant") %>
+
+**rsCtrlAuxH=*choice***
+
+Type of auxiliary heating control.
+
+<%= csv_table(<<END, :row_header => true)
+Choice, Description
+C_AUXHEATCTRL_LO, Compressor locked out if any auxiliary heating control
```
Comment (nealkruis) [inline doc/src/records/rsys.md:573]: Convert to user input choices.

**Review by nealkruis — CHANGES_REQUESTED** [review]

File: doc/src/records/report.md (hunk 12 above)
Comment (galanCA) [reply doc/src/records/report.md:199]: It is if-out should I remove the if statement and replace it, or should I keep it how it is right now?

File: doc/src/records/dhwsys.md [hunk 16]
```diff
@@ -212,7 +388,7 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
 
 **wsTSetPoint=*float***
 
-  Specifies the hot water setpoint temperature for all child DHWHEATERs.  Used only for HPWH-based DHWHEATERs (HPWH models tank temperatures and heating controls), otherwise has no effect.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:215]: This shouldn't have changed. I think your spell checker doesn't recognize "setpoint". You should add it to your dictionary.

File: doc/src/records/dhwsys.md [hunk 17]
```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
   ---------- ---------------------- ---------------------- -------------- -----------------
               *name of a DHWDAYUSE*   (no scheduled draws)        No             daily
 
-**wsFaucetWaste=*float***\
-**wsShowerWaste=*float***\
-**wsBathWaste=*float***\
-**wsCWashrWaste=*float***\
-**wsDWashrWaste=*float***
+**wsUnkDrawDurF=*float***
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:158]: Let's move Unknown to the end of all the end uses (since it's the exception case).

File: doc/src/records/dhwsys.md [hunk 18]
```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
+<%= member_table(
+  units: "gal/draw",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsShowerDrawDurF=*float***
+
+Draw duration factor for shower fixture. This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:182]: Similar change here (and the rest of the end uses).

File: doc/src/records/dhwsys.md [hunk 19]
```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
+<%= member_table(
+  units: "gal/draw",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsCWashrDrawDurF=*float***
+
+Draw duration factor for clothing washer fixture. This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:204]: "clothes washer", not "clothing washer".

File: doc/src/records/dhwsys.md [hunk 20]
```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
+Draw duration factor for dishwasher fixture. This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
+
+<%= member_table(
+  units: "gal/draw",
+  legal_range: "x $\\geq$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsUnkDrawWaste=*float***
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:224]: Move to end of ws*DrawWaste inputs.

File: doc/src/records/dhwsys.md [hunk 21]
```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
+
+Draw water waste for unknown fixture.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:226]: Pluralize fixtures for all of these inputs.

File: doc/src/records/dhwsys.md [hunk 22]
```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
+Draw water waste for dish washer fixture.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
 
 **wsBranchModel=*choice***
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:290]: This one needs documentation still (I think chip just left it as "ToDo").

File: doc/src/records/dhwsys.md [hunk 23]
```diff
@@ -186,21 +312,71 @@ Day waste scaling factor.
   ----------- ----------------- ------------------------------ -------------- -----------------
      --        $\ge$ 0             1                               No            constant
 
+**wsUnkDayWasteF=*float***
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:315]: Move to end of "ws*DayWasteF" inputs.

File: doc/src/records/dhwsys.md [hunk 24]
```diff
@@ -186,21 +312,71 @@ Day waste scaling factor.
...
 
+Unknown relative water draw for day of waste scheme.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:317]: The CNRECS comment isn't very clear here. Maybe @chipbarnaby can come up with better description text?

File: doc/src/records/dhwsys.md [hunk 25]
```diff
@@ -221,12 +397,100 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
...
+<%= member_table(
+  units: "",
+  legal_range: "x $>$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsDRMethod=*choice***
+
+Response control method for domestic hot water demand.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:452]: Demand response control method.

See this [description of Demand Response](https://en.wikipedia.org/wiki/Demand_response#:~:text=Demand%20response%20is%20a%20reduction,peak%20and%20occasional%20demand%20spikes.).

File: doc/src/records/dhwsys.md [hunk 26]
```diff
@@ -221,12 +397,100 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
...
+  legal_range: "See table above",
+  default: "NONE",
+  required: "No",
+  variability: "At the start of a run") %>
+
+**wsDRSignal=*choice***
+
+Response control signal for domestic hot water demand.
+
+<%= csv_table(<<END, :row_header => true)
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:472]: We should add descriptions for these choices too. @chipbarnaby or @pmskintner, we might need your help here since there isn't anything in the source code (that I could find).

File: doc/src/records/dhwsys.md [hunk 27]
```diff
@@ -283,6 +547,28 @@ Specifies electrical parasitic power to represent recirculation pumps or other s
...
+<%= member_table(
+  units: "Hr",
+  legal_range: "x $\\geq$ 0",
+  default: "4",
+  required: "No",
+  variability: "constant") %>
+
+**wsLoadMaxDur=*integer***
+
+Maximum load duration for the window.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:563]: Same as above.

File: doc/src/records/gain.md [hunk 28]
```diff
@@ -125,6 +138,17 @@ Name of a DHWSYS whose water use modulates gnPower.  For example, electricity us
 ----------- ------------------- ---------------------- -------------- -----------------
             *name of a DHWSYS*  no DHWSYS/GAIN linkage            No         constant
 
+**gnCtrlDHWMETER=*integer***
+
+Controls the DHWMETER. Allows gains to track water usage such as dishwashers, clothes washers, etc.
```
Comment (nealkruis) [inline doc/src/records/gain.md:143]: This is actually the DHWMETER that controls the gain (not the other way around as it currently reads).

File: doc/src/records/rsys.md [hunk 29]
```diff
@@ -136,6 +136,10 @@ Specifies type of motor driving the fan (blower). This is used in the derivation
   ----------- ----------------- ------------- -------------- -----------------
               PSC, BPM          PSC           No             constant
 
+**rsAdjForFanHt=*NOYESCH***
```
Comment (nealkruis) [inline doc/src/records/rsys.md:139]: I think this is just ***choice***. Look for other examples. I could be wrong.

File: doc/src/records/rsys.md [hunk 30]
```diff
@@ -274,6 +591,17 @@ For rsType=ASHP, auxiliary electric (“strip”) heating capacity. If AUTOSIZEd
   ----------- --------------------------- ------------- -------------- -----------------
   Btu/hr      *AUTOSIZE* or *x* $\ge$ 0   0             no             constant
 
+**rsAFUEAuxH=*float***
+
+For rsType=ASHP, auxiliary electric (“strip”) heating coefficient of performance. Energy use for auxiliary heat is accumulated to end use HPBU of meter rsElecMtr (that is, auxiliary heat is assumed to be electric).
```
Comment (nealkruis) [inline doc/src/records/rsys.md:596]: Replace "coefficient of performance" with "annualized fuel utilization efficiency"

**Review by nealkruis — CHANGES_REQUESTED** [review]

File: doc/src/records/dhwsys.md (hunk 26 above)
Comment (pmskintner) [reply doc/src/records/dhwsys.md:472]: Here's some documentation on the commands:

ON - Normal operation following the water heater's internal control logic.
TOO - Tops of the tank once by engaging the all the available heating sources (compressor and resistive elements) in the water heater to heat the tank to setpoint, regardless of the current condition.
TOOLOR - Tops of the tank once and locks out the resistance elements, just the compressor is used to heat the tank to setpoint. 
TOOLOC- Tops of the tank once and locks out the compressor, just the resistance elements are used to heat the tank to setpoint. 
TOT - Tops of the tank on a timer using all the available heating sources (compressor and resistive elements) in the water heater. The tank starts a timer and heating to setpoint when the call is received, and repeats the heating call when the timer reaches zero.
TOTLOR - Tops of the tank on a timer and locks out the resistance elements, just the compressor is used to heat the tank to setpoint. 
TOTLOC - Tops of the tank on a timer and locks out the compressor, just the resistance elements is used to heat the tank to setpoint. 
LOC - Locks out the compressor from the water heater's normal internal control logic.
LOR - Locks out the resistive elements from the water heater's normal internal control logic.
LOCLOR  - Locks out the compressor and resistive elements from the water heater's normal internal control logic.
COUNT - Not sure on this one... @chipbarnaby?

I believe the setpoint can also be input as a schedule, i.e. Choice

File: doc/src/records/dhwsys.md [hunk 31]
```diff
@@ -283,6 +547,28 @@ Specifies electrical parasitic power to represent recirculation pumps or other s
 ----------- ------------------ ------------- --------------- -----------------
    W            $\ge$ 0             0             No             hourly
 
+**wsDrawMaxDur=*integer***
+
+Maximum draw duration for the window.
```
[the remark this answers is not in the document]
Comment (pmskintner) [reply doc/src/records/dhwsys.md:552]: I think this is unrelated to the Ecosizer implementation, but would most likely be used for sizing following the ASHRAE simple methodology.

File: doc/src/dhwheater-doc.md [hunk 32]
```diff
@@ -0,0 +1,466 @@
...
+"GenericTier3","Generic Tier 3"
+"UEF2Generic","Experimental UEF=2"
+"BasicIntegrated","Typical integrated HPWH"
+"ResTank","Resistance heater (no compressor).  Superceded by whHeatSrc=RESITANCEX"
+"ResTankNoUA","Resistance heater (no compressor) with no tank losses.  Superseded by whHeatSrc=RESISTANCEX."
+"AOSmithHPTU80DR","80 gallon AOSmith HPTU with fixed backup setpoint (experimental for demand response testing)"
+"AOSmithSHPT50","50 gal AOSmith SHPT"
+"AOSmithSHPT66","66 gal AOSmith SHPT"
+"AOSmithSHPT80","80 gal AOSmith SHPT"
+END
```
Comment (pmskintner) [inline doc/src/dhwheater-doc.md:299]: There is a bunch of new choices here, check CNDTYPES.def, for all of the models.

File: doc/src/records/rsys.md (hunk 29 above)
Comment (galanCA) [reply doc/src/records/rsys.md:139]: I think I saw other examples like that. If I see them again I'll change them to choice.

File: doc/src/records/export.md [hunk 33]
```diff
@@ -101,6 +105,17 @@ Specifies DHW meter(s) whose data is to be exported, for *exType*=DHWMTR.
   ----------- ----------------------------- ------------- ------------------------------ -----------------
               name of a *DHWMETER*, ALL, SUM              Required for *exType*=DHWMTR   constant
 
+**exAFMeter=*integer***
```
Comment (nealkruis) [inline doc/src/records/export.md:108]: Should be `afMtrName`.

File: doc/src/records/gain.md [hunk 34]
```diff
@@ -125,6 +138,17 @@ Name of a DHWSYS whose water use modulates gnPower.  For example, electricity us
 ----------- ------------------- ---------------------- -------------- -----------------
             *name of a DHWSYS*  no DHWSYS/GAIN linkage            No         constant
 
+**gnCtrlDHWMETER=*integer***
```
Comment (nealkruis) [inline doc/src/records/gain.md:141]: Should be `dhwMtrName`.

File: doc/src/records/top-members.md [hunk 35]
```diff
@@ -813,6 +817,17 @@ Deprecated method for specifying cooling autosizing days.  Design conditions are
      months                         *none*           No           constant
 
 
+**coolDsCond=*integer***
```
Comment (nealkruis) [inline doc/src/records/top-members.md:820]: Should be `descondName` (DESCOND document also needs to be created).

File: doc/src/dhwheater-doc.md [hunk 36]
```diff
@@ -284,18 +289,70 @@ Air source heat pump type, valid only if whHeatSrc=ASHPX. These choices are supp
...
+"NyleC250A_MP","Nyle Cxx external HPWHs (MP = multi-pass mode)"
+"NyleC60A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC90A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC125A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC185A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC250A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"Rheem_HPHD60HNU_MP","???"
+"Rheem_HPHD60VNU_MP","???"
+"Rheem_HPHD135HNU_MP","???"
+"Rheem_HPHD135VNU_MP","???"
```
Comment (nealkruis) [inline doc/src/dhwheater-doc.md:353]: @pmskintner @chipbarnaby Are these used? Can we get descriptions for them?

Comment (pmskintner) [reply doc/src/dhwheater-doc.md:353]: To keep the same format as other descriptions, something like: "Rheem HPHDxx MP external MP HPWHs"

Comment (chipbarnaby) [reply doc/src/dhwheater-doc.md:353]: Agree with Paul, this list has to be trued up with CNDTYPES.DEF.  The Rheem models are defined, but I would defer to Paul as to how to describe them.

File: doc/src/cse-user-manual.yaml [hunk 37]
```diff
@@ -28,13 +28,32 @@ html-site-top: "/index.html"
...
   - [1, "introduction.md"]
   - [1, "operation.md"]
   - [1, "input-structure.md"]
   - [1, "input-data.md"]
   - [2, "records/top-members.md"]
   - [2, "records/holiday.md"]
+  - [2, "records/descond.md"]
+<% if inactive_CNE_records %>
+  - [2, "records/inverse.md"]
+<% end %>
```
Comment (nealkruis) [inline doc/src/cse-user-manual.yaml:56]: This isn't needed. I don't think we'll ever need this in the documentation. It sounds like it was a half-developed feature that won't be used. Right, @chipbarnaby?

File: doc/src/records/airhandler.md [hunk 38]
```diff
@@ -1536,6 +1536,22 @@ METER to record crankcase heater energy use, category "Aux"; not recorded if not
...
+**ahccSHR**
+
+**ahccVfRperTon**
+
+**coilOversize**
+
+**fanOversize**
+
+**pydxCaptFLim**
+
```
Comment (nealkruis) [inline doc/src/records/airhandler.md:1554]: Let's flesh these out.

File: doc/src/records/boiler.md [hunk 39]
```diff
@@ -124,6 +124,14 @@ The following four allow specification of meters to record boiler auxiliary ener
...
 **blrAuxFullOffMtr=*mtrName***
 
+**blrAuxFullOff=*deprecated***
+
+**blrAuxOff=*deprecated***
+
+**blrAuxOn=*deprecated***
+
+**blrAuxOnAtall=*deprecated***
+
```
Comment (nealkruis) [inline doc/src/records/boiler.md:134]: Let's flesh these out.

File: doc/src/records/coolplant.md [hunk 40]
```diff
@@ -69,7 +69,17 @@ TOWERPLANT that cools the condenser water for the chillers in this COOLPLANT.
...
+
+**cpStage3=ALL**
+
+**cpStage4=ALL**
+
+**cpStage5=ALL**
+
+**cpStage6=ALL**
+
+**cpStage7=ALL**
```
Comment (nealkruis) [inline doc/src/records/coolplant.md:82]: Let's flesh these out.

File: doc/src/records/descond.md [hunk 41]
```diff
@@ -0,0 +1,124 @@
...
+Design dry-bulb temp.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "no-limitations?",
+  default: "0.0",
+  required: "No",
+  variability: "constant") %>
+
+**dcDay=*DOY***
```
Comment (nealkruis) [inline doc/src/records/descond.md:16]: "DOY" is not a valid type for the documentation.

File: doc/src/records/dhwsys.md [hunk 42]
```diff
@@ -457,7 +457,7 @@ Excess size factor for domestic hot water design. wsFxDes is applied when wsHeat
 
 **wsDRMethod=*choice***
 
-Demand response control method. Use to better match the power demand with the supply, in order to reduce peak demand.
+Used to better match the power demand with the supply, in order to reduce peak demand.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:460]: We talked about removing this sentence and keeping the other sentence.

File: doc/src/records/heatplant.md [hunk 43]
```diff
@@ -50,7 +50,17 @@ Heat plant pipe loss: heat assumed lost from piping connecting boilers to loads
...
+
+**hpStage3=ALL**
+
+**hpStage4=ALL**
+
+**hpStage5=ALL**
+
+**hpStage6=ALL**
+
+**hpStage7=ALL**
```
Comment (nealkruis) [inline doc/src/records/heatplant.md:63]: Let's flesh this out.

File: doc/src/records/rsys.md [hunk 44]
```diff
@@ -897,6 +897,14 @@ Air dry-bulb temperature at the outdoor portion of this system.
...
   ^o^F                          From weather file   No             hourly
 
+**rsSHRtarget=*float***
+
+Temporary test
+
+**rsSrcSideLoadMtr=*float***
+
+Temporary test
+
```
Comment (nealkruis) [inline doc/src/records/rsys.md:907]: What is "Temporary test"?

File: doc/src/records/terminal.md [hunk 45]
```diff
@@ -381,6 +381,10 @@ Name of meter, if any, which is to record energy used by this terminal fan. The
   ----------- ------------------- ---------------- -------------- -----------------
               *name of a METER*   *not recorded*   No             constant
 
+**tuOversize=*deprecated***
+
+**tuVfMxHC=*deprecated***
+
```
Comment (nealkruis) [inline doc/src/records/terminal.md:387]: Let's flesh this out.

**Comment by nealkruis** [comment]
The other question I have is if we can move dhwheater-doc.md into the records sub-folder.

File: doc/src/records/dhwsys.md [hunk 46]
```diff
@@ -221,12 +405,100 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
...
+TOOLOR, Tops of the tank once and locks out the resistance elements, just the compressor is used to heat the tank to setpoint.
+TOOLOC, Tops of the tank once and locks out the compressor, just the resistance elements are used to heat the tank to setpoint.
+TOT, Tops of the tank on a timer using all the available heating sources (compressor and resistive elements) in the water heater. The tank starts a timer and heating to setpoint when the call is received, and repeats the heating call when the timer reaches zero.
+TOTLOR, Tops of the tank on a timer and locks out the resistance elements, just the compressor is used to heat the tank to setpoint.
+TOTLOC, Tops of the tank on a timer and locks out the compressor, just the resistance elements is used to heat the tank to setpoint.
+LOC, Locks out the compressor from the water heater's normal internal control logic.
+LOR, Locks out the resistive elements from the water heater's normal internal control logic.
+LOCLOR, Locks out the compressor and resistive elements from the water heater's normal internal control logic.
+COUNT, ???
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:492]: This needs to be dropped. All enums end in COUNT, but it doesn't correlate to user input.

**Review by nealkruis — CHANGES_REQUESTED** [review]

File: doc/src/records/report.md (hunk 12 above)
Comment (nealkruis) [reply doc/src/records/report.md:199]: Similarly here. What is the status on this thread?

File: doc/src/records/dhwsys.md [hunk 47]
```diff
@@ -144,26 +155,149 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
...
 
-**wsFaucetWaste=*float***\
-**wsShowerWaste=*float***\
-**wsBathWaste=*float***\
-**wsCWashrWaste=*float***\
-**wsDWashrWaste=*float***
+**wsFaucetDrawDurF=*float***
 
-Specifies additional draw volume per DHWUSE event (at fixture, by end use).  This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
+Draw duration factor for faucets. This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
```
Comment (nealkruis) [inline doc/src/records/dhwsys.md:160]: It looks like the content for `ws*Waste` was copied to `ws*DrawDurF` by accident.

**Review by nealkruis — APPROVED** [review]
This is looking good to me. @chipbarnaby can you take a pass through?
